### ISSUE TITLE
some question about yolov3

### ISSUE THREAD

**Issue by PythonImageDeveloper** [experiencor/keras-yolo3#2]
For abjectness scores it use logistic regression . what is this advantage ? in the yolo2 what is use for abjectness scores? why this train multi-label classification?  what was in the v2? multi-class classification ? what is advantage of this ?

**Comment by experiencor** [comment]
Objectness varies from 0 to 1, right? What else to use other than logistic regression.

The author just wants to train a multi-label classification. So he uses logistic regression and squared loss instead of cross entropy.

**Comment by xiaohu2015** [comment]
@experiencor Did you haved tested the pretrained model and get some detection images? Did you get correct coords?

**Comment by zeynali** [comment]
@experiencor , `Objectness varies from 0 to 1, right? What else to use other than logistic regression.`
right , but the author in the paper said in the work (v3) we use  logistic regression , and in the v2 was Objectness for boxes , right ? so my question is that in the v2 what else to use for Objectness. in my opinion in the v3 for calculate Objectness different from v2 , right?

**Comment by experiencor** [comment]
1. The 2 versions are slightly different in treating objectness ground truth. In v2, it's set to IOU of predicted box and ground truth box. In v3, it's simply set to 1. As a result, I find that v3 produces relatively high objectness score for a detection and you can use 0.5 as a threshold. In v3, you must one some value like 0.3 in order to obtain good number of detections.

2. They are different in many small details but the major one is that v3 uses 3 heads to predict bounding at 3 scales: 13x13, 26x26, 52x52. It's better at picking up small objects.

3. What you mean by result?

**Comment by zeynali** [comment]
my mean is mAP and speed (FPS) for your own dataset or pascal voc /COCO.

**Comment by experiencor** [comment]
I'm training it on COCO. Will tell you the result when it's finished.
